Emby.Server.Implementations/TV/TVSeriesManager.cs

```diff
@@ -66,7 +66,7 @@ public QueryResult<BaseItem> GetNextUp(NextUpQuery query, DtoOptions options)
             else
             {
                 parents = _libraryManager.GetUserRootFolder().GetChildren(user, true)
-                   .Where(i => i is Folder)
+                   .Where(i => i is CollectionFolder f && f.CollectionType == CollectionType.tvshows)
```

Comment:
Can "mixed movies and tv" show up in next up?
If so this collection type filter should include those libraries